src/include/miopen/conv/problem_description.hpp

```diff
@@ -235,6 +235,15 @@ struct ProblemDescription
     std::size_t GetBackwardPadW() const { return GetWeightsWidth() - GetPadW() - 1; }
     std::size_t GetBackwardPadH() const { return GetWeightsHeight() - GetPadW() - 1; }
 
+    bool IsAsymmetricPadH() const
+    {
+        return conv.paddingMode == miopenPaddingSame && (GetWeightsHeight() % 2) == 0;
```

Comment:
Please check code around https://github.com/ROCmSoftwarePlatform/MIOpen/blob/cb1ef1f2a89ebbc4e556cce51234456d9dc9ea03/src/convolution.cpp#L242-L248

It seems like in certain cases we can get asymmetric padding even in miopenPaddingDefault mode.

Reply:
2 * GetConvPads()[i] - is symmetric padding;
1 - case where filter bigger than input image;